doc/man7/provider-mac.pod

```diff
@@ -153,7 +155,8 @@ parameters are relevant to, or are understood by all macs:
 
 =item "key" (B<OSSL_MAC_PARAM_KEY>) <octet string>
 
-Sets the key in the associated MAC ctx.
+Sets the key in the associated MAC ctx.  This is identical to passing a I<key>
+argument to the EVP_MAC_init() function.
```

Comment:
If we're talking in terms of this manual page, shouldn't that refer to `OSSL_FUNC_mac_init()`?

Reply:
Yep.  Changed.